src/dndbeyond/content-scripts/character.js

```diff
@@ -326,8 +326,7 @@ function handleSpecialMeleeAttacks(damages=[], damage_types=[], properties, sett
 
     // Feats
     // Great Weapon Master Feat
-    if (to_hit !== null && 
-        character.getSetting("great-weapon-master", false) &&
+    if (character.getSetting("great-weapon-master", false) &&
```

Comment:
why did you remove the check for to_hit in here?

Reply:
There is no condition under which there isn't a `to_hit` on this now in the function call